<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件伪造器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/ico/img/Logo.ico</normaloff>:/ico/img/Logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
</string>
        </property>
        <property name="title">
         <string>文件路径</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>请输入文件地址,支持拖放</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>选择文件</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/ico/img/autoAdd.ico</normaloff>:/ico/img/autoAdd.ico</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 154, 214)</string>
          </property>
          <property name="text">
           <string>仅支持伪造zip文件</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(220, 0, 0)</string>
          </property>
          <property name="text">
           <string>重要文件请不要使用此软件!!!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>使用教程</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/ico/img/aboutFHF.ico</normaloff>:/ico/img/aboutFHF.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>伪造或还原</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/ico/img/execute.ico</normaloff>:/ico/img/execute.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actionb"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionabout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ico/img/Logo.ico</normaloff>:/ico/img/Logo.ico</iconset>
   </property>
   <property name="text">
    <string>关于本程序</string>
   </property>
   <property name="iconText">
    <string>关于本程序</string>
   </property>
  </action>
  <action name="actionb">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ico/img/qt-logo.png</normaloff>:/ico/img/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
